examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java

```diff
@@ -106,11 +107,23 @@ public void onClick() {
     EditText thirdValueInput = findViewById(R.id.thirdValueInput);
     values.add(Float.parseFloat(thirdValueInput.getText().toString()));
 
+    // TODO(#988): Move blocking call ouf of UI thread.
     submitSample(bucket, indices, values);
   }
 
-  // TODO(#717): Use Java version of gRPC without invoking JNI.
-  private native void createChannel(String address);
+  /** Gets the custom CA certificate from the assets folder. */
+  private String getCaCertificate() {
+    try {
+      return new BufferedReader(new InputStreamReader(getAssets().open("ca.pem")))
```

Comment:
Does it mean that the client will use only `ca.pem` and not try to use OS certificates?

Reply:
Yes.